bootloader/src/old_machine_bootloader_linux.c

```diff
@@ -90,15 +50,16 @@ size_t writefunc(void *ptr, size_t size, size_t nmemb, struct response *s)
 
 char* executeCommand(char* commandLine){
     FILE *fp;
-    char* path = (char *) malloc(sizeof(char) * 300);
+    const int maxOutputLength = 2400;
```

Comment:
Why 2400?

Reply:
I don't see how we can asses the length of command output apart from guesstimating. For the commands used in this program 2400 is more than enough.

Reply:
If it's a constant allocation, can we hoist it out?

Reply:
added realloc